packages/client/src/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from "clsx"
+import { twMerge } from "tailwind-merge"
+ 
+export function cn(...inputs: ClassValue[]) {
```

Comment:
I am not familiar with this util function, what is the purpose of using a wrapper like this?

Reply:
https://github.com/kir-dev/startlap-sch/commit/0f634c684d8ba9b8d9d441bbd266682c99f4eba2#diff-4efe1827e56ec6e9806b48b76785b5bae80950d0c3eb32c87dff1101b4cd364e
This wasnt added by us.